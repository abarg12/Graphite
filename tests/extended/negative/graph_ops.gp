graph<int, tree> g1;
graph<int, tree> g2;
graph<int, tree> g3;

g3 = g1 union g2; 

int funky() {
    graph<int, tree, other> g2;
    return 1;
}
