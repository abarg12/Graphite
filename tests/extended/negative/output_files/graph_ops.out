Fatal error: exception Failure("invariant 'other' does not exist")
